Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Merchant.hs

```diff
@@ -47,3 +50,11 @@ type Merchant = MerchantD 'Safe
 instance FromJSON (MerchantD 'Unsafe)
 
 instance ToJSON (MerchantD 'Unsafe)
+
+data Slot = Slot
+  { startTime :: TimeOfDay,
+    endTime :: TimeOfDay,
+    slot :: Text
```

Comment:
Why slot needs to be text? It can be number, right?

Reply:
I was using S1, S2 as slots thats why I gave Text. Will change it to number.